spec/rubocop/config_loader_spec.rb

```diff
@@ -246,6 +246,19 @@
           .to be_superset(expected.to_set)
       end
     end
+
+    context 'when a file inherits from an expanded path' do
+      let(:file_path) { '.rubocop.yml' }
+
+      before do
+        create_file('~/.rubocop.yml', [''])
```

Comment:
Hmm, I tested this on my local machine and no rubocop file is located in my home directory. Rather, if I print the path for this file, the file is created in `/private/var/folders/......./home/.rubocop.yml`, which is in `tmp` so it should be cleaned up. It seems like this should be fine, right?